bin/pulumilocal

```diff
@@ -124,22 +193,79 @@ def set_localstack_pulumi_config(args: argparse.Namespace):
 def main():
     # Parse arguments from call to pulumi CLI that set the stack name and directory
     parser = argparse.ArgumentParser(add_help=False)
-    parser.add_argument("command", help="pulumi primary command",
+    parser.add_argument("command", help="Pulumi command",
                         type=str)
-    parser.add_argument("-s", "--stack", help="pulumi stack name",
+    parser.add_argument("-s", "--stack", help="The name of the stack to operate on. Defaults to the current stack",
                         required=False,
                         type=str)
-    parser.add_argument("-C", "--cwd", help="run in this directory",
+    parser.add_argument("-C", "--cwd", help="Run pulumi as if it had been started in another directory",
                         required=False,
                         type=str)
+    parser.add_argument("--config-file", help="Use the configuration values in the specified file rather than detecting the file name",
+                        required=False,
+                        type=str)
+    parser.add_argument("--non-interactive", help="Disable interactive mode for all commands",
+                        action="store_true",
+                        required=False)
     args, _ = parser.parse_known_args()
 
+    if NON_INTERACTIVE:
+        args.non_interactive = True
+
+    if not args.cwd:
+        args.cwd = "."
+
+    if os.environ.get("CONFIG_STRATEGY") not in CONFIG_STRATEGIES:
+        print(f"Config strategy: {os.environ.get('CONFIG_STRATEGY')}")
+        print(f'Config strategy is not recognised. Falling back to default ("{DEFAULT_CONFIG_STRATEGY}").')
```

Comment:
This is a duplicate from the global `CONFIG_STRATEGY`. The error message here is better, though these messages should probably print to stderr.

Also: is this a fatal error, or just a warning? If the former then we should exit the program. Perhaps that would be clearer to the user?

```suggestion
    if os.environ.get("CONFIG_STRATEGY") not in CONFIG_STRATEGIES:
        print(f"Config strategy: {os.environ.get('CONFIG_STRATEGY')}", file=sys.stderr)
        print(f'Config strategy is not recognised. Falling back to default ("{DEFAULT_CONFIG_STRATEGY}").', file=sys.stderr)
```

Reply:
Technically it's not an error more of a warning. Wanted to keep the luxury for the users that they don't need to reconfigure anything if they use the new version.
With your proposal any previous user's CI would start to fail with the new version.

Reply:
As per above, if I understand the change correctly, if users don't specify a `CONFIG_STRATEGY` then it will fall back to the current behaviour, so this would not break if we errored.

It is personal preference, but if I configure a tool incorrectly, I would prefer to know about it when I run it, not for it to fall back to a default with a log message which I may miss. I don't read, it turns out... 😂